infra/experimental/chronos/prepare-ccache

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/bash
+# Copyright 2024 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+################################################################################
+
+PROJECT=$1
+{
+    echo "COPY ccache-cache/ /ccache/cache";
+    echo "ENV COMPILE_USING_CCACHE=1"
```

Comment:
We can just set `PATH` to `/ccache/bin:$PATH` here instead of introducing another variable right?

Reply:
yup, good catch, fixed.